ios/PPSSPP-Info.plist

```diff
@@ -202,5 +202,11 @@
 			</dict>
 		</dict>
 	</array>
+	<key>NSBonjourServices</key>
```

Comment:
I think this needs a dummy `ALTDeviceID` plist key in order to enable JIT automatically

https://discord.com/channels/625766896230334465/626109028824514570/882747590679076895

Reply:
Updated